core/ajax.py

```diff
@@ -861,6 +881,37 @@ def update_metadata(self, imdbid, tmdbid=None):
         else:
             return r
 
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def movie_metadata(self, imdbid, tmdbid=None, language=None):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)